src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/CreateApplicationTaskHandler.java

```diff
@@ -128,7 +128,8 @@ private void withoutNoticeNoConsent(GeneralApplication generalApplication, CaseD
          * Add the GA in respondent one collection if he/she initiate without notice application.
          * */
 
-        if (generalApplication.getGeneralAppApplnSolicitor().getOrganisationIdentifier()
+        if (generalApplication.getGeneralAppApplnSolicitor().getOrganisationIdentifier() != null
```

Comment:
It was not null during my testing, but that may be because some earlier set up assumes there will be a solicitor, and if that were to be fixed then yes this would break. I have added the extra null check.